core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -360,7 +360,10 @@ int findVersion() {
    */
   private void renameToFinal(FileSystem fs, Path src, Path dst, int nextVersion) {
     try {
-      lockManager.acquire(dst.toString(), src.toString());
+      boolean success = lockManager.acquire(dst.toString(), src.toString());
+      if (!success) {
+        throw new CommitFailedException("Failed to acquire lock on file: %s", dst);
+      }
```

Comment:
Style nit, if we could add a new line after this `if` block separating it from the next `if`! I know in this file it looks like we didn't really follow our normal practice already but at least for new changes we could follow it.

Also I think I'd just inline the boolean so

```
if (!lockManager.acquire(dst.toString(), src.toStrnig()) {
 throw ...
}
```

Reply:
Also should we include the owner (src) in the exception as well?